server/src/models/Project/project.controller.ts

```diff
@@ -21,25 +20,26 @@ export class ProjectController {
     }
   }
 
-  async createProject (req: Request, res: Response): Promise<void> {
+  async createProject(req: Request, res: Response): Promise<void> {
     try {
-
-      const { user_id } = req.params
-      const project = await service.createProject(+user_id, req.body)
+      const { user_id } = req.body;
+      const id = user_id;
+      const { title, description, status } = req.body;
```

Comment:
should we instead of destructuring it twice, only pass the req.body in the function?

Reply:
createProject will complain because it's expecting 2 arguments